integration/apigateway-domain/index.ts

```diff
@@ -0,0 +1,80 @@
+import * as aws from '@pulumi/aws';
+import * as pulumicdk from '@pulumi/cdk';
+import { RestApi } from './rest-api';
+import { ResourceAttributeMappingArray, ResourceMapping } from '../../lib/interop';
+
+class ApiGatewayStack extends pulumicdk.Stack {
+    constructor(id: string, options?: pulumicdk.StackOptions) {
+        super(id, options);
+        this.node.setContext('@aws-cdk/aws-apigateway:disableCloudWatchRole', 'true');
+
+        new RestApi(this, 'test-api');
+        this.synth();
+    }
+}
+
+const stack = new ApiGatewayStack('test-api-domain', {
+    remapCloudControlResource: (logicalId, typeName, props, options): ResourceMapping | undefined => {
+        if (typeName === 'AWS::CertificateManager::Certificate') {
+            const resources: ResourceAttributeMappingArray = [];
+            const hostedZoneId: string = props.DomainValidationOptions[0].HostedZoneId;
+            const certLogicalId = `${logicalId}-cert`;
```

Comment:
Does this mean the user can pick anything into logicalId to report in the ResourceMapping? Just making up unique strings?

Reply:
Yep that's right. The logicalId is only important for references. So if you create a `Certificate` and then another resources references that, we do that by logicalId.

Reply:
It sounds like it's the logical ID implied by some CF notion of logical ID. Naming is hard.